File: lenggiauit-portfolio-api/Resources/MockInterviewResource.cs
using System;

namespace Lenggiauit.API.Resources
{
    public class MockInterviewResource
    {
        public Guid Id { get; set; }
        public string FullName { get; set; }
        public string Email { get; set; }
        public string AgeRange { get; set; }
        public string Language { get; set; }
        public string Resume { get; set; }
        public string CoverLetter { get; set; }
        public string JobDescription { get; set; }
        public string Note { get; set; }
       
        public Guid? EventBookingDateId { get; set; }
        public EventBookingDateResource EventBookingDate { get; set; }
        public EventRequestChangeReasonResource EventRequestChangeReason { get; set; }
        public string EventStatus { get; set; }
        public bool IsEnableRequestChange { get; set; }
        public bool IsEnableDelete { get; set; }
        public string RedeemCode { get; set; }
        public string Code { get; set; }
    }
}

File: lenggiauit-portfolio/src/services/resources/mockInterviewResource.ts
import { EventBookingDateResource } from "./eventBookingDateResource";
import { EventRequestChangeReasonResource } from "./eventRequestChangeReasonResource";

export type MockInterviewResource =
{
    id  : any,
    fullName  : any, 
    email  : any, 
    ageRange  : any, 
    fullname: any, 
    language: any,
    resume: any,
    coverLetter: any,
    jobDescription: any,  
    note: any,
    eventBookingDateId: any
    eventBookingDate? : EventBookingDateResource,  
    eventStatus  : any,
    isEnableRequestChange: boolean,
    isEnableDelete: boolean,
    eventRequestChangeReason: EventRequestChangeReasonResource
    code: any,
    redeemCode: any,
}